cluster.tf

```diff
@@ -324,8 +324,9 @@ resource "google_container_node_pool" "pools" {
   dynamic "autoscaling" {
     for_each = lookup(each.value, "autoscaling", true) ? [each.value] : []
     content {
-      min_node_count = lookup(autoscaling.value, "min_count", 1)
-      max_node_count = lookup(autoscaling.value, "max_count", 100)
+      location_policy = "BALANCED"
```

Comment:
We should make the change in https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/tree/master/autogen and run `make build`. It would also be good to expose as an option to make it user configurable.